hkd/hkd.cabal

```diff
@@ -0,0 +1,61 @@
+cabal-version: 2.2
+name: hkd
+version: 0
+synopsis: "higher-kinded data"
+description:
+  "Higher-kinded data" utilities.
+homepage: https://github.com/ekmett/codex/tree/master/hkd#readme
```

Comment:
Presumably, this will change?

Reply:
Yes. As implied, I didn't put too much work into human/documentation side of things yet.